core/src/main/java/org/elasticsearch/action/admin/cluster/node/info/NodesInfoResponse.java

```diff
@@ -73,12 +73,6 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 builder.byteSizeField("total_indexing_buffer", "total_indexing_buffer_in_bytes", nodeInfo.getTotalIndexingBuffer());
             }
 
-            if (nodeInfo.getServiceAttributes() != null) {
```

Comment:
I guess this piece of info that is not returned by nodes info was already returned in some other section of nodes info as well? I think we have a proper `http` section for it that is still there?